lib/sqlite3.js

```diff
@@ -366,6 +368,18 @@ SQLite3.prototype._buildColumnDefinition = function(model, propertyName) {
   return line;
 };
 
+SQLite3.prototype._isPrimaryKeyInteger = function(model) {
+  var properties = this.getModelDefinition(model).properties;
+  for (var propertyName in properties) {
+    if (!properties.hasOwnProperty(propertyName)) continue;
+    var property = this.getModelDefinition(model).properties[propertyName];
+    if (property.id && this._columnDataType(model, propertyName) === 'TEXT') {
+      return false;
+    }
+  }
+  return true;
```

Comment:
```suggestion
  var properties = this.getModelDefinition(model).properties;
  for (var propertyName in properties) {
    if (!properties.hasOwnProperty(propertyName)) continue;
    var property = this.getModelDefinition(model).properties[propertyName];
    if (property.id) {
      return this._columnDataType(model, propertyName) === 'INTEGER';
    }
  }
  return false;
```
This function should return true only the type of id is `INTEGER`, and the loop should be broken if we found the id property, :)

Reply:
Our code doesn't it handle composite id, but it's even not supported in this connector yet ...